website/src/welcomeMessage.tsx

```diff
@@ -0,0 +1,4 @@
+const welcomeMessage = `# ModMark
+Hello and welcome to the ModMark playground!
+`;
+export default welcomeMessage;
```

Comment:
Can't we use the previous document as a welcome message? It includes some more stuff to play around with, and was easy for us to handle since it was in a separate file and such